pages/nft/url/index.vue

```diff
@@ -290,28 +328,29 @@ export default class FetchIndex extends Vue {
       )
       if (arweaveId) { this.arweaveId = arweaveId }
       this.ipfsHash = ipfsHash
-      return {
+      this.arweaveFeeInfo = {
         to: address,
         amount: new BigNumber(LIKE),
         memo,
       }
     } catch (err) {
       // eslint-disable-next-line no-console
-      console.error('CANNOT_ESTIMATE_ARWEAVE_FEE')
-      throw err
+      console.error(err)
+      throw new Error('CANNOT_ESTIMATE_ARWEAVE_FEE')
     }
   }
 
-  async sendArweaveFeeTx({ to, amount, memo }: { to: string, amount: BigNumber, memo: string }): Promise<void> {
+  async sendArweaveFeeTx(): Promise<void> {
+    const { to, amount, memo } = this.arweaveFeeInfo
```

Comment:
Personally I think it's more readable if you read and write `arweaveFeeInfo` in the `doAction` loop, instead of the new change here

Reply:
Yes, I agree. But currently, the read/write of the original `arweaveFeeInfo` happens in different case blocks, so I think I have to store it in prop temporarily.

Reply:
Where do you use `sendArweaveFeeTx` except in `doAction`? If we only use it in do action, I think the `this.arweaveFeeInfo` can happen in doAction instead of here